helper/helper-common/src/main/java/com/linkedin/avroutil1/compatibility/AvroAdapter.java

```diff
@@ -134,10 +136,14 @@ default FieldBuilder newFieldBuilder(String name) {
 
   void setFieldPropFromJsonString(Schema.Field field, String propName, String valueAsJsonLiteral, boolean strict);
 
+  boolean sameJsonProperties(Schema.Field a, Schema.Field b, boolean compareStringProps, boolean compareNonStringProps);
+
   String getSchemaPropAsJsonString(Schema schema, String propName);
 
   void setSchemaPropFromJsonString(Schema field, String propName, String valueAsJsonLiteral, boolean strict);
 
+  boolean sameJsonProperties(Schema a, Schema b, boolean compareStringProps, boolean compareNonStringProps);
+
```

Comment:
why do we need two methods here?

Reply:
one for schemas, the other for fields. they dont share a common parent class across all avro versions